components/DeductionPopup/DeductionPopup.module.css

```
@@ -19,6 +45,13 @@
   overflow-y: auto;
   position: relative;
   animation: slide-in 0.3s ease-out;
+  overflow: auto;
```

Stylelint (17.12.0)

[error] Overridden property "overflow-y" by shorthand "overflow" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)